dogs.html

```diff
@@ -0,0 +1,130 @@
+        </div>
+        <a class="adopt-btn" href="#">Adopt</a>
+      </div>
+
+      <div class="doggo-card">
+        <img class="card-img" src="images/poppy-card.jpg" alt="">
+        <div class="card-text">
+          <h4>Poppy</h4>
+          <p><strong>Cost to Adopt: </strong>$123.45</p>
+          <p>Most angery pupper I have ever seen pupperino many pats fat boi, boof waggy wags. What a nice floof doggorino clouds blep he made many woofs woofer big ol pupper.</p>
+        </div>
+        <a class="adopt-btn" href="#">Adopt</a>
+      </div>
+
+      <div class="doggo-card">
+        <img class="card-img" src="images/jack-card.jpg" alt="">
+        <div class="card-text">
+          <h4>Jack</h4>
+          <p><strong>Cost to Adopt: </strong>$123.45</p>
+          <p>Most angery pupper I have ever seen pupperino many pats fat boi, boof waggy wags. What a nice floof doggorino clouds blep he made many woofs woofer big ol pupper.</p>
+        </div>
+        <a class="adopt-btn" href="#">Adopt</a>
+      </div>
+
+      <div class="doggo-card">
+        <img class="card-img" src="images/duffy-card.jpg" alt="">
+        <div class="card-text">
+          <h4>Duffy</h4>
+          <p><strong>Cost to Adopt: </strong>$123.45</p>
+          <p>Most angery pupper I have ever seen pupperino many pats fat boi, boof waggy wags. What a nice floof doggorino clouds blep he made many woofs woofer big ol pupper.</p>
+        </div>
+        <a class="adopt-btn" href="#">Adopt</a>
+      </div>
+
+      <div class="doggo-card">
+        <img class="card-img" src="images/lucas-card.jpg" alt="">
+        <div class="card-text">
+          <h4>Lucas</h4>
+          <p><strong>Cost to Adopt: </strong>$123.45</p>
+          <p>Most angery pupper I have ever seen pupperino many pats fat boi, boof waggy wags. What a nice floof doggorino clouds blep he made many woofs woofer big ol pupper.</p>
+        </div>
+        <a class="adopt-btn" href="#">Adopt</a>
+      </div>
+
+      <div class="doggo-card">
+        <img class="card-img" src="images/jake-card.jpg" alt="">
+        <div class="card-text">
+          <h4>Jake</h4>
+          <p><strong>Cost to Adopt: </strong>$123.45</p>
+          <p>Most angery pupper I have ever seen pupperino many pats fat boi, boof waggy wags. What a nice floof doggorino clouds blep he made many woofs woofer big ol pupper.</p>
+        </div>
+        <a class="adopt-btn" href="#">Adopt</a>
+      </div>
+
+      <div class="doggo-card">
+        <img class="card-img" src="images/angus-card.jpg" alt="">
+        <div class="card-text">
+          <h4>Angus</h4>
+          <p><strong>Cost to Adopt: </strong>$123.45</p>
+          <p>Most angery pupper I have ever seen pupperino many pats fat boi, boof waggy wags. What a nice floof doggorino clouds blep he made many woofs woofer big ol pupper.</p>
+        </div>
+        <a class="adopt-btn" href="#">Adopt</a>
```

Comment:
Tip, these will be buttons that do JS, not a navigation link

Reply:
Good to know. I will replace this with a button.